libs/community/tests/unit_tests/vectorstores/test_sqlserver.py

```diff
@@ -292,3 +295,22 @@ def test_sqlserver_delete_text_by_id_no_ids_provided(
     # Should return False, since empty list of ids given
     if not result:
         pass
+
+
+def test_that_multiple_vector_stores_can_be_created(
+    store: SQLServer_VectorStore,
```

Comment:
does mentioning a function name as the argument, invoke it as well?

Reply:
Yes.